scripts/create-release-notes.ts

```diff
@@ -0,0 +1,81 @@
+import * as shell from 'shelljs';
+const changeLogGeneratorUtils = require('./change-log-generator-utils');
+const constants = require('./change-log-constants');
+
+const [_, __, releaseOverride] = process.argv
+
+const logger = (msg: string, obj?: any) => {
+  if (!obj) {
+    console.log(`*** ${msg}`);
+  } else {
+    console.log(`*** ${msg}`, obj);
+  }
+};
+
+/**
+ * Checks if the user has provided a release branch override. If they
+ * have not, return the latest release branch.
+ */
+function getCurrentRemoteReleaseBranch(): string {
+  logger('\nStep 1: Determine release branch.');
+  let releaseBranch;
+  if (!releaseOverride) {
+    console.log(`releaseOverride not provided. Getting latest release.`);
```

Comment:
you made an awesome logger then used console.log :P

Reply:
hahahaha I know I know - but I decided I didn't always love the extra *** at the start of the line, and liked it more to separate steps. I do need to add a logger to change-log-generator-utils for that at least to be consistent though.